### PR TITLE
Create page end game

### PR DESCRIPTION
**Pull request by MASBERNATQ**

### CODE REVIEW

**Comment by FlavianEng** [comment]
Task linked: [CU-860qfe5ch Create page end game](https://app.clickup.com/t/860qfe5ch)

File: src/shared/Modal/components/Modal.tsx [hunk 1]
```diff
@@ -58,12 +59,14 @@ const Modal = ({
     <>
       <BehindOverlay isVisible={isVisible} onClose={handleClose} />
       <div className={modalClassName} data-testid="Modal">
-        <Header
-          className="Modal__Header"
-          leftSideNode={leftSideNode}
-          rightSideNode={rightSideNode}
-          title={title}
-        />
+        {!hideHeader && (
```
Comment (FlavianEng) [inline src/shared/Modal/components/Modal.tsx:62]: It's a bad practice to prioritise negation, you can improve this by renaming your property with "showHeader" and male it positive

Comment (MASBERNATQ) [reply src/shared/Modal/components/Modal.tsx]: ✅

File: src/game/components/EndGameRecap.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  );
+
+  return (
+    <Modal className="EndGameRecap" isVisible showHeader={false}>
+      <Title title={titleMessage} type="h3" className="EndGameRecap__Title" />
+
+      <PlayersCards />
+
+      <div className="EndGameRecap__buttons">
+        <Button size="large" style="secondary" onClick={onLeave} iconName="Logout">
```
Comment (FlavianEng) [inline src/game/components/EndGameRecap.tsx:35]: This icon name seems misnamed, "logout" stands for account disconnection, like sign out

**Review by FlavianEng — APPROVED** [review]

File: src/game/components/EndGameRecap.tsx (hunk 2 above)
Comment (MASBERNATQ) [reply src/game/components/EndGameRecap.tsx:35]: ✅

**Review by FlavianEng — DISMISSED** [review]

**Review by FlavianEng — APPROVED** [review]

**Pull request merged**